### PR TITLE
revised communication page

### PR DESCRIPTION
**Pull request by jodygarnett**
revised with notes on responsible disclosure and a copy of the geotools issue reporting instructions (without pictures)

### CODE REVIEW

File: comm/index.html [hunk 1]
```diff
@@ -84,9 +85,85 @@
...
+            </ul>
+        </li>
+        <li>What Happens Next?
+            <p>On creation a notification will automatically be sent to the geoserver-devel list. JIRA sends notifications for everything done on the issue, to the reporter, the assignee, and to anyone who clicks on the link to ‘watch’ an issue.</p>
+            <p>This is why you must sign up for an account, so that JIRA can email you when updates are done. Your email will not be used for anything else. One nice little feature of JIRA is that if you reply to the email sent for notification, then the reply will show up as a comment on the issue.</p>
+        </li>
+        <li>For New Bugs we may need assistance to reproduce the issue before a fix can be considered.
+          <p>Usually the a developer will need to ask you for more information. Until they can reproduce you issue, or you volunteer to test, not much is going to happen.</p>
+          <p>Example: If you are on some exotic hardware (like oracle on windows server) that we do not have public access to you will probably need to arrange to meet on IRC and test out different solutions with a developer.</p>
```
Comment (aaime) [inline comm/index.html]: "exotic setup" maybe? Oracle and Windows are not hardware

File: comm/index.html [hunk 2]
```diff
@@ -84,9 +85,85 @@
...
+        </li>
+        <li>
+           Please keep in mind that many members of the GeoServer team have a day job. Just as you volunteer your time to report an issue they volunteer their time to support the project.
+        </li>
+      </ol>
+      <p>
+      As GeoServer is open source, everyone is encouraged to fix bugs and submit patches. Even if you are not a core developer, we welcome patches <a href="https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md">contributed</a> through GitHub <a href="https://github.com/geoserver/geoserver/pulls">pull requests</a>, or attached to a JIRA issue.
+      </p>
+    </div>
+    <div class="alert alert-warning">
```
Comment (aaime) [inline comm/index.html]: Maybe push this one up? I fear we might have already lost our reader here, and this one is important.

**Comment by aaime** [comment]
Looks good minus a couple of small issues

File: comm/index.html [hunk 3]
```diff
@@ -84,9 +85,85 @@
...
         </ul>
+       <p> The
+            same <a href="userlist-guidelines.html" class="alert-link">post guidelines</a> as the user list apply to the developer list.</p> 
     </section>
 </section>
 
+<section>
+    <h2>Issue Tracker</h2>
+    <p>
+      If you have a problem when working with GeoServer, then please let us know through the mailing lists. GeoServer uses <a href="https://osgeo-org.atlassian.net/projects/GEOS">JIRA</a>, a bug tracking website, to manage issue reports.
```
Comment (jodygarnett) [inline comm/index.html]: I wanted to ask about "please let us know through the mailing lists". 

Was that intentional to cut down on duplicate bug reports? Or should we say issue tracker there...

Comment (aaime) [inline comm/index.html]: Very intentional, the bug tracker is often abused as a way to ask questions, or as a way to report an issue that is not a GeoServer problem, but a user error or misunderstanding of some kind, making Jira maintenace just harder. So, we should really stress that a ticket should be opened when one is _sure_  it's a bug.

**Pull request merged**